### PR TITLE
Added workshop`s rating for short model called by StatisticController

### PR DESCRIPTION
**Pull request by BHy4uk**

### CODE REVIEW

**Review by fkma13552 — APPROVED** [review]

File: OutOfSchool/OutOfSchool.WebApi.Tests/Services/StatisticServiceTest.cs [hunk 1]
```diff
@@ -22,24 +23,30 @@ public class StatisticServiceTest
 
         private Mock<IApplicationRepository> applicationRepository;
         private Mock<IWorkshopRepository> workshopRepository;
+        private Mock<IRatingService> ratingService;
```
Comment (VadymLevkovskyi) [inline OutOfSchool/OutOfSchool.WebApi.Tests/Services/StatisticServiceTest.cs:26]: `ratingService` seems to be not ever used outside `SetUp` method. Thus might be moved inside it as local var

Comment (VadymLevkovskyi) [reply OutOfSchool/OutOfSchool.WebApi.Tests/Services/StatisticServiceTest.cs:26]: Same for `logger`

Comment (VadymLevkovskyi) [reply OutOfSchool/OutOfSchool.WebApi.Tests/Services/StatisticServiceTest.cs:26]: Same for `mapper`

**Review by VadymLevkovskyi — APPROVED** [review]
There small issues with tests, might be fixed in other PRs

**Review by DmyMi — APPROVED** [review]

**Review by toherberg — APPROVED** [review]

**Pull request merged**